docs/tiger_data_spec.md

```diff
@@ -133,3 +136,113 @@ A typical search will return records in the following format when using ElasticS
 
 }
 ```
+
+## Data creation
+
+The census geocoder uses Elasticsearch synonyms to resolve abbreviations (i.e. St for Street, or MD for Maryland).
+The synonyms.txt file with the synonyms definition must be installed in every node in the cluster, in the same directory as the elasticsearch.yml configuration file.
+
+* First, create the index, with the synonyms analyzer settings, and apply that analyzer to the corresponding fields:
```

Comment:
Based on earlier chat ...
Do we need to create the index first if we can update it as needed?
Would doing it this way require an update to the loader?

See this gist - https://gist.github.com/awolfe76/cd411fe198810ff873bc

In there I was able to update the create the `analyzer` and then update the `mapping` to use it **after** the index had been created.

*Worked on both census and point data. Also tested STATE on census with ME = Maine.*